CHANGES.md

```diff
@@ -1,5 +1,39 @@
 # Changes
 
+## edge-21.2.2
+
+This edge release introduces a new `repair` command which will repopulate
+resources needed for upgrading from a `2.9.x` installation. There will be an
+error message during the upgrade process indicating that this command should be
+run so that users do not need to guess.
+
+It also continues to prepare support for multicluster opaque ports.
```

Comment:
We're underselling this. TCP multicluster support is a huge deal. That it doesn't yet work with opaque ports is a small implementation detail that we can comment on in the next release. Prior to this release, multicluster only supported HTTP. Now it can work with any protocol!!! Even without an opaque-ports annotation. The opaque-ports stuff just makes it _better_. This is the top-line feature in this edge release.

Reply:
Yep that's my bad. I was only thinking about this with opaque ports in mind. I included this at the top and focused more on the TCP support.